packages/mgt/src/styles/fabric-icon-font.ts

```diff
@@ -5,6 +5,8 @@
  * -------------------------------------------------------------------------------------------
```

Comment:
Should we rename this file now that it's more than the font?

Reply:
Probably! I will do that. Good catch.